docs/xdr/features/collect/integrations/cloud_and_saas/aws/aws_flow_logs.md

```diff
@@ -28,19 +28,53 @@ For VPC and subnet:
 - Click on *Create flow log*
 - Set up the flow log: we recommend to capture all traffic (accepted and rejected).
 
+### Create a SQS queue
+
+This integration relies on S3 Event Notifications to discover new S3 objects.
```

Comment:
```suggestion
This integration relies on S3 Event Notifications (SQS) to discover new S3 objects.
```

Introduce the "SQS" term instead of "S3 Event Notifications" as you use it later in your documentation